js_modules/dagit/packages/core/src/runs/RunDetails.tsx

```diff
@@ -116,20 +120,49 @@ export const RunConfigDialog: React.FC<{run: RunFragment; isJob: boolean}> = ({r
   const [showDialog, setShowDialog] = React.useState(false);
   const {rootServerURI} = React.useContext(AppContext);
   const runConfigYaml = yaml.stringify(run.runConfig) || '';
+
   return (
     <div>
       <Group direction="row" spacing={8}>
+        <Button
+          icon={<Icon name="edit" />}
+          onClick={() =>
+            window.open(
+              workspacePathFromRunDetails({
+                id: run.id,
+                repositoryName: run.repositoryOrigin?.repositoryName,
+                repositoryLocationName: run.repositoryOrigin?.repositoryLocationName,
+                pipelineName: run.pipelineName,
+                isJob,
+              }),
+            )
+          }
+        >
+          Open in Launchpad
+        </Button>
```

Comment:
I think it would be okay to use `<AnchorButton to={...}>` here and just open it in the same window.